firmware/ch32x035-usb-device-compositekm-c/sdk/Debug/debug.c

```diff
@@ -10,8 +10,17 @@
 * Attention: This software (modified or not) and binary are used for
 * microcontroller manufactured by Nanjing Qinheng Microelectronics.
 *******************************************************************************/
+
 #include "debug.h"
 
+#include "ch32x035.h"
+
+#include "ch32x035_gpio.h"
+#include "ch32x035_rcc.h"
+#include "ch32x035_usart.h"
+
+#include "system_ch32x035.h"
```

Comment:
This include is in `User/`, because it refers to `SystemCoreClock`, in `DelayInit`.